### DISCUSSION TITLE
Local I-analyzer installation for personal corpora

### DISCUSSION THREAD

**Discussion by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#1440] [Ideas]
Especially useful when we add the option for users to add corpora: enable students/researchers to install I-analyzer locally and add their own corpora. Especially useful for data that should not be part of the public interface like social media data or copyrighted material.

**Comment by BeritJanssen** [comment]
This should be relatively easy using Docker.

**Comment by lukavdplas** [comment]
Indeed, though the majority of our target audience uses Windows and would not be helped by installation instructions that start with installing WSL :sweat_smile:
